docs/UserGuide.md

```diff
@@ -45,27 +45,45 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 **:information_source: Notes about the command format:**<br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+  e.g. in `ingredient NAME`, `NAME` is a parameter which can be used as `ingredient milk`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+  e.g `NAME [TAG]` can be used as `John Doe friend` or as `John Doe`.
 
 * Items with `…`​ after them can be used multiple times including zero times.<br>
   e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
 
 * Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+  e.g. if the command specifies `NAME PHONE_NUMBER`, `PHONE_NUMBER NAME` is also acceptable.
 
 </div>
 
 ### Viewing help : `help`
 
-Shows a message explaning how to access the help page.
-
-![help message](images/helpMessage.png)
+Prints the list of commands, including the name, format and purpose of each command.
 
 Format: `help`
 
+### Ingredients Tracking
+#### List Ingredient Levels : `ingredient`
+* Prints the ingredient levels for all ingredient types retrieved from the database.
+
+Format: `ingredient list`
+
+* Prints the ingredient level for a particular type of ingredient which is specified by the user’s command.
+
+Format: ` ingredient single INGREDIENT_NAME`
```

Comment:
there is a white space before "ingredient", I think it should be a typo and need to be removed?

Reply:
thanks, resolved